vhdl_ls/src/stdio_server.rs

```
@@ -64,24 +78,39 @@ pub fn start() {
         let stdin = io::stdin();
         loop {
             let request = read_request(&mut stdin.lock());
-            request_sender.send(request).unwrap();
+            match request_sender.send(request) {
+                Ok(_) => continue,
+                Err(_) => break,
```

Maybe add an eprintln! of the error to not silently swallow it